content/events/2020-london/sponsor.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Sponsor
+---
+
+Kubernetes Community Days is a community-organized, not-for-profit conference series for practitioners.
+
+Sponsoring Kubernetes Community Days events gives your organization an opportunity to speak with practitioners, managers, and executives from companies of all sizes and industries. Whether your goals include recruiting new talent, opening new markets, or connecting with your existing customers, this event will be the place to have those conversations with experts and leaders in tech innovation.
+
+To enquire about sponsoring Kubernetes Community Days London 2020 get in touch via email at [sponsor@kubernetesdays.london](mailto:sponsor@kubernetesdays.london).
```

Comment:
Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here? 

We are also looking at creating another alias sponsor-london@kubernetescommunitydays.org as well so this can stay consistent in format. Thank you.